hw_2.js

```diff
@@ -0,0 +1,135 @@
+"use strict"
+
+// Task 1 
+
+let x = 20;
+let y = 58;
+let z = 42;
+
+console.log(x + y + z);
+
+
+// Task 2
+
+const SEC_IN_MIN = 60;
+const MIN_IN_HR = 60;
+const HR_IN_DAY = 24;
+const DAY_IN_YR = 365;
+let yearsNumber = 53; //полных лет
+let daysNumber = 110; //дней в неполном году - д.р. 10.01, сегодня 30.04
+const ageInSeconds = (yearsNumber * DAY_IN_YR + daysNumber) * HR_IN_DAY * MIN_IN_HR * SEC_IN_MIN
+
+console.log(ageInSeconds);
+
+
+// Task 3
+
+let count = 42; // число
+
+console.log(count);
+console.log(typeof count); 
+
+// // число на строку 1й вариант
+count = String(count);
+console.log(typeof count);
+
+// // число на строку 2й вариант
+let countString = "count";              // новая переменная
+console.log(typeof countString); 
+
+
+let userName = '42'; // строка
+
+console.log(userName);
+console.log(typeof userName);
+
+// // строка на число 1й вариант
+userName = Number(userName);
+console.log(typeof userName);
+
+// // строка на число 2й вариант
+let userNameNumber = Number(userName);  // новая переменная
+console.log(typeof userNameNumber);
+
+
+// Task 4
+
+let a = 1;
+let b = 2;
+let c = "белых медведей";
+
+a = String(a);
+b = String(b);
+console.log(a, b, c);
```

Comment:
так тоже можно
НОсмысл сделать через соединение строк
```suggestion
console.log(a, b, c);
```